crossgen.cmake

```diff
@@ -4,6 +4,7 @@ add_definitions(
     -DCROSSGEN_COMPILE
     -DCROSS_COMPILE
     -DFEATURE_NATIVE_IMAGE_GENERATION
+    -DFEATURE_JIT_METHOD_PERF
```

Comment:
My bad, should have tried more build flavors; sorry. I have no idea why this caused problems, but FWIW the reason I added the thing here is that I saw other - similar looking - options already there so it seemed like the logical place. If switches.h is a more appropriate place for this flag, though, that is where I'll move it. Thanks for the info, @jkotas .